scripts/prepare_chunks.sh

```diff
@@ -32,23 +32,27 @@ for GAM_FILE in "${GAM_FILES[@]}"; do
     # Put gam files into array format
     if [ -z "$GAM_FILES_JSON" ]
     then
-        GAM_FILES_JSON="$GAM_FILE"
+        GAM_FILES_JSON="[$GAM_FILES_JSON\"$GAM_FILE\""
     else
-        GAM_FILES_JSON="$GAM_FILES_JSON, $GAM_FILE"
+        GAM_FILES_JSON="$GAM_FILES_JSON, \"$GAM_FILE\""
     fi
 
     vg_chunk_params=" $vg_chunk_params -a $GAM_FILE"
 done
 
+GAM_FILES_JSON="$GAM_FILES_JSON]"
+
+echo $GAM_FILES_JSON
 
```

Comment:
It looks like `jq` can convert a file of lines to a JSON array of strings in a way that will properly handle special characters:

```
$ printf 'abc\n123\n",drop table students,"\n' | jq -R '[.]' | jq -n '[inputs[]]'
[
  "abc",
  "123",
  "\",drop table students,\""
]
```

So you might want to try that instead of hand-building JSON in Bash?

You can grab a temporary file with something like `TEMP_FILE_NAME="$(mktemp)"`